pkg/errors/errors.go

```diff
@@ -86,6 +89,9 @@ var (
 
 	//ErrDivertNotSupported raised if the divert feature is not supported in the current cluster
 	ErrDivertNotSupported = fmt.Errorf("the 'divert' field is only supported in namespaces managed by Okteto")
+
+	//ContextIsNotOktetoCluster raised if the cluster connected is not managed by okteto
+	ContextIsNotOktetoCluster = fmt.Errorf("okteto context is not managed by okteto")
```

Comment:
What's the difference with?
https://github.com/okteto/okteto/pull/1757/files#diff-ab01ccb939ab896a48c5445d1d755bf861ad2f337d98013132834c6d88641c8aR52